bin/dfx-sns-sale-finalize

```diff
@@ -18,7 +18,9 @@ source "$(clap.build)"
 
 dfx canister --network "$DFX_NETWORK" call sns_swap finalize_swap '(record {})' | tee ,sns-sale-finalize.idl
 
-if idl2json <,sns-sale-finalize.idl | jq 'to_entries | map({key: .key, value: {failure: .value[0].failure}} | select((.value.failure != 0) and (.value.failure != null)))| from_entries | select (. != {})' | grep .; then
+ERROR_MESSAGE="$(idl2json <,sns-sale-finalize.idl | jq -r '.error_message')"
```

Comment:
This too could go out of date, so we need to continue to be vigilant.  But thank you for updating the check!

Reply:
Yes, but if this goes out of date, the script might fail via `set -e` (if the `.error_message` field no longer exists).
The previous check being out of date, just caused the `if` to branch, resulting in success.